src/tests/test_integration.py

```diff
@@ -46,7 +46,9 @@ def test_integration_basic():
     try:
         entities = load_data()
         assert len(entities) > 1
-        check_data(entities)
+        # it seems data consolidation takes now longer in cratedb
+        time.sleep(20)
```

Comment:
To be honest, I haven't checked if the issue is the notification from orion or the consolidation from crate.
I tend to think the second, because with crate 4.1.x we use shorter time.

Reply:
oh dear, something we'll need to think about in prod when we upgrade...

Reply:
Dear Federico and Andrea,

while I am not into the details here, maybe this advise might help you along. Apologies upfront if this does not apply here.

In general you need to refresh a table [1] after inserting some data in order to make data consolidate/synchronize immediately. If that applies to the code here, I am still wondering why it would take 20 seconds, as the default refresh interval is set to 1000 milliseconds.

In any case, we usually invoke `REFRESH TABLE foobar;` within our integration test scenarios after inserting data into the database [2,3]. Otherwise, the test suite would fail more often than not.

With kind regards,
Andreas.

[1] https://crate.io/docs/crate/reference/en/4.5/general/dql/refresh.html
[2] https://github.com/crate/crate-qa/blob/6630834/tests/client_tests/python/asyncpg/test_asyncpg.py#L38-L42
[3] https://github.com/crate/crate-qa/blob/6630834/tests/client_tests/node-postgres/tests/test_insert_multivalue.js#L15-L19

Reply:
thx @amotl the test invoked works for both crate and timescaledb, so that's why we don't apply a refresh to crate, probably something clever can be done to force it only for cratedb tests. but i am not sure it is worth. on ci-circle we would just save 20 secs over 8 minutes :)

Reply:
but even if we don't refresh explicitly just waiting 1 second after the last insert should be enough right? 

> default refresh interval is set to 1000 milliseconds

or am I misinterpreting the above statement?